proto/validatorstate/validator_state.proto

```diff
@@ -40,12 +43,32 @@ message GetValidatorSetRequest {
   bytes subnet_id = 2;
 }
 
+message GetCurrentValidatorSetRequest {
+  bytes subnet_id = 1;
+}
+
 message Validator {
   bytes node_id = 1;
   uint64 weight = 2;
   bytes public_key = 3;
 }
 
+message CurrentValidator {
```

Comment:
Possibly dumb question but is there a good reason for not adding new fields to the existing `Validator` message instead of adding a new one? i.e. do we see the data returned by them becoming incompatible at some point? It seems like it should be compatible since protobuf fields are all optional

Reply:
It would mean a breaking change. Changing `Validator` would require changes to related functions + clients. I think if we want to we can deprecate old methods and types in favor of `CurrentValidator`. @StephenButtolph

Reply:
Why not just have a field of type `Validator` in `CurrentValidator`? then you don't need to copy the first three fields.

Reply:
Well I think it wasn't that much painful as I thought it would be. I was under impression that I also had to change `GetValidatorSet` return types, but turned out we can just leave unused fields empty for that function and keep the return type same.